CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 3.5.0 (unreleased)
+
+NEW FEATURES:
+
+* `random_bytes` generates an array of random bytes that is intended to be used as key or secret ([#63](https://github.com/hashicorp/terraform-provider-random/issues/63))
+
```

Comment:
```suggestion
```

We have recently switched to using [Changie](https://changie.dev/) for changelog and version management.

Could you remove this entry from CHANGELOG.md and once Changie is [installed](https://changie.dev/guide/installation/) run the following:

```shell
changie new
? Kind: Features
Body: resource/random_bytes: New resource that generates an array of random bytes intended to be used as key or secret ([#63](https://github.com/hashicorp/terraform-provider-random/issues/63)).
Issue/PR Number: 272
```